middleware/crud-functions/crud-function-tests/postLosersTest.js

```diff
@@ -8,6 +8,22 @@ describe('postLosers', function() {
     revertLossData(['nash']);
     response.should.equal(true);
   });
+
+  it("should increment losses, winStreak and set lossStreak to 0", async function() {
+    const beforeUser = await getUser("nash");
+    await postWinners(["nash"]);
+    const afterUser = await getUser("nash");
+
+    assert.equal(afterUser["losses"], beforeUser["losses"]+1);
+    assert.equal(afterUser["lossStreak"], beforeUser["lossStreak"]+1);
+    assert.equal(afterUser["lossStreak"], 0);
+    revertWinData(["nash"]);
```

Comment:
Use `afterEach()` so we don't have to retype `revert` for every test.

Reply:
we pass in a value which might change so imo its better to keep it explicit so we don't assume that the data will revert if we have different winners in a new test